src/types.ts

```diff
@@ -1,4 +1,11 @@
+/*
+	This file contains definitions of some interfaces and classes that are used in Source (such as
+	error-related classes).
+*/
+
+/* tslint:disable:interface-functionName max-classes-per-file */
```

Comment:
It's slightly strange that you had to disable this lint for this file even though it appears that this file didn't change. Was this line added because you made a change in this file and later reverted it?

Reply:
You're about right. I added the comment block at the top to help myself understand what was going on when I was still figuring things out on my own. During a merge I decided to keep that block there, while also accepting incoming changes (the comment that disabled the linter). At the time I figured there'd be no harm in leaving a one-line comment in the code. That said, what would you suggest now? Should I remove line 6?

Reply:
You're about right. I added the comment block at the top to help myself understand what was going on when I was still figuring things out on my own. During a merge I decided to keep that block there, while also accepting incoming changes (the comment that disabled the linter). At the time I figured there'd be no harm in leaving a one-line comment in the code. That said, what would you suggest now? Should I remove line 6?

Reply:
Yes. You can view the blame for this file to see where these lines come from. (either view it on github or use `git blame`)

Reply:
It seems to have come from #175 ? Either way, I'll remove it.